### ISSUE TITLE
Emoji completion in bio

### ISSUE THREAD

**Issue by SamuelLongchamps** [tuskyapp/Tusky#2845]
I cannot seem to trigger emoji completion when updating a profile's bio or display name. Given the feature works in posts, I assumed it might be a limitation of the input control used there.

* * * *
- Tusky Version: 19.0
- Android Version: 13
- Android Device: Pixel 6
- Mastodon instance (if applicable): fosstodon

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by kylegoetz** [comment]
Yes, it's not currently in the code. According to the Mastodon creator, they're a (regrettably, in his opinion) permitted thing in usernames, so I'll work on adding the functionality.

https://mastodon.social/@Gargron/105148594404369117

>Sometimes I wonder if allowing emoji in display names in the first place was a mistake 😕
